third_party/terraform/resources/resource_bigquery_table.go

```diff
@@ -683,6 +694,26 @@ func resourceBigQueryTableRead(d *schema.ResourceData, meta interface{}) error {
 			return err
 		}
 
+		edc := map[string]interface{}(nil)
```

Comment:
This gave a lint error:
```
==> Checking source code against linters...
google/resource_bigquery_table.go:697:3: ineffectual assignment to `edc` (ineffassign)
		edc := map[string]interface{}(nil)
		^
make: *** [lint] Error 1
GNUmakefile:26: recipe for target 'lint' failed
```

since edc isn't used outside of the if statement below, you can just `:=` it on line 711.

Reply:
Fixed